sdk/synapse/azure-analytics-synapse-monitoring/CHANGELOG.md

```diff
@@ -1,15 +1,23 @@
 # Release History
 
 ## 1.0.0-beta.4 (2021-08-10)
```

Comment:
For unreleased version, do we need an entry stating the azure-json migration? (also, for the migration of other synapse-libs )

Reply:
Would be good to add once this is released again, but I didn't as that may not be for a while (if ever)